compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/PropertyReferenceLowering.kt

```diff
@@ -0,0 +1,347 @@
+/*
+ * Copyright 2019 Google LLC. Use of this source code is governed by the Apache 2.0 license
```

Comment:
According to our [license](https://github.com/JetBrains/kotlin/tree/master/license), all code in this repository is copyright by JetBrains. Therefore we need our default copyright on all sources. Is there any particular reason why you've changed this to Google LLC?